pkgs/top-level/all-packages.nix

```diff
@@ -14358,7 +14358,7 @@ in
 
   mariadb = callPackage ../servers/sql/mariadb {
     asio = asio_1_10;
-    jemalloc = jemalloc.override ({ disableInitExecTls = true; });
+    jemalloc = jemalloc450.override ({ disableInitExecTls = true; });
```

Comment:
There is probably a reason for this change, it would be good to add a comment that states this reason.

Reply:
fixed this error:
```
-- Looking for malloc_stats_print in jemalloc_pic
-- Looking for malloc_stats_print in jemalloc_pic - found
-- Performing Test JEMALLOC_OK
-- Performing Test JEMALLOC_OK - Failed
CMake Error at storage/tokudb/CMakeLists.txt:63 (MESSAGE):
  static jemalloc_pic.a can only be used up to jemalloc 4

-- Configuring incomplete, errors occurred!
See also "/build/mariadb-10.3.15/build/CMakeFiles/CMakeOutput.log".
See also "/build/mariadb-10.3.15/build/CMakeFiles/CMakeError.log".
```

Reply:
Sounds good, let's explain it in the code with a comment like

```
# As per mariadb's cmake, "static jemalloc_pic.a can only be used up to jemalloc 4".
```

so that when `jemalloc450` disappears some time in the future, the person who does it knows what's going on.